### ISSUE TITLE
Prototype TTS engine for audio voiceover failover

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#4406]

**Comment by BenHenning** [comment]
Technical reference: https://developer.android.com/reference/android/speech/tts/TextToSpeech.
